dev/custom-pages/social.js

```diff
@@ -32,15 +32,4 @@
     }
   });
 
-  $(document).on('DOMNodeInserted', function(e) {
```

Comment:
Does line 4 not need to be updated still to fix the twitter feed affixing?  (Change `#footer` to `.site-footer`)